source/server/config/http/fault.cc

```diff
@@ -0,0 +1,71 @@
+#include "envoy/http/header_map.h"
+#include "envoy/server/instance.h"
+
+#include "common/common/empty_string.h"
+#include "common/http/filter/fault_filter.h"
+#include "server/config/network/http_connection_manager.h"
+
+namespace Server {
+namespace Configuration {
+
+/**
+ * Config registration for the fault injection filter. @see HttpFilterConfigFactory.
+ */
+class FaultFilterConfig : public HttpFilterConfigFactory {
+public:
+  HttpFilterFactoryCb tryCreateFilterFactory(HttpFilterType type, const std::string& name,
+                                             const Json::Object& json_config,
+                                             const std::string& stats_prefix,
+                                             Server::Instance& server) override {
+    if (type != HttpFilterType::Decoder || name != "fault") {
+      return nullptr;
+    }
+
+    /**
```

Comment:
to clarify.. are you saying use only the // comments inside functions and not the multiline comment syntax?